inc/saivlan.h

```diff
@@ -386,6 +386,19 @@ typedef enum _sai_vlan_attr_t
      */
     SAI_VLAN_ATTR_BROADCAST_FLOOD_GROUP,
 
+    /**
+     * @brief Enable/Disable ARP & NDP suppression
+     *
+     * When enabled, the ARP and NDP messages are trapped to CPU instead of
+     * getting copied (and flooding the vlan). Default is set to false
+     * for backward compatibility reasons
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_VLAN_ATTR_NEIGH_SUPPRESSION_ENABLE,
```

Comment:
why not use full "NEIGHBOR" instead NEIGH?

Reply:
Thanks Kamil, I changed the macro name.